detox/src/devices/android/APKPath.test.js

```diff
@@ -12,10 +12,9 @@ describe('APKPath', () => {
   });
 
   it(`path for a gradle build flavor`, async () => {
-    const inputApkPath = '~/somePath/build/outputs/apk/development/debug/app-development-debug.apk';
```

Comment:
Please add tests rather than replacing, we need to make sure all variations work.

Reply:
@rotemmiz good call, done